storage/metrics_storage.go

```diff
@@ -27,6 +27,11 @@ import (
 
 	"github.com/hawkular/hawkular-openshift-agent/config"
 	"github.com/hawkular/hawkular-openshift-agent/log"
+	"github.com/hawkular/hawkular-openshift-agent/k8s"
+	"time"
+	"strings"
+	"github.com/hawkular/hawkular-openshift-agent/config/security"
+	"os"
```

Comment:
not to be finicky (ok, I'm being finicky) -- HOSA imports follow the pattern that all system imports are at the top, followed by blank line, followed by third party imports, followed by blank line, followed by HOSA imports. Putting in the blank lines and running gofmt will order the imports appropriately.

So "time", "strings", "os" go at the top, and the two hawkular imports go at the bottom.

I know of no way to enforce this via gofmt, so its just something to manually do.